docs/tutorials/01-intro.md

```diff
@@ -7,6 +7,11 @@ The purpose of this tutorial is to demonstrate how Buildah can be used to build
 
 In brief the `containers/image` project provides mechanisms to copy, push, pull, inspect and sign container images. The `containers/storage` project provides mechanisms for storing filesystem layers, container images, and containers. Buildah is a CLI that takes advantage of these underlying projects and therefore allows you to build, move, and manage container images and containers.
 
+Note that installation instructions assume you are running a Linux distro that uses `dnf` as its package manager, and have all prerequisites fulfilled. See Buildah's [installation instructions][buildah-install] for a full list of prerequisites, and the `buildah` installation section in the [official Red Hat documentation][rh-repo-docs] for RHEL-specific instructions.
```

Comment:
Suggest:

Note that the installation instructions below assume

Reply:
I didn't realize markdown would working with a double bracket like this for a link.  I thought it had to be bracket and parenthesis like [installation instructions](buildah-install), but this seems to work great, something learned!